addon/components/bsy-radio-buttons/button.js

```diff
@@ -1,22 +1,21 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { isEqual } from '@ember/utils';
 import layout from '../../templates/components/bsy-radio-buttons/button';
 
 const BsyRadioButtonsButtonComponent = Component.extend({
   layout,
   tagName: 'label',
   classNames: 'btn btn-secondary',
-  classNameBindings: ['activeClassName'],
+  classNameBindings: ['activeClassName:active'],
 
-  activeClassName: computed('paramValue', function () {
-    if (this.get('paramValue') === this.get('option')) {
-      return 'active';
-    }
-  })
+  activeClassName: computed('value', function() {
```

Comment:
aren;t we observing option here?

Reply:
That's what I thought as well but that doesn't change, right?